next.config.js

```diff
@@ -1,7 +1,18 @@
+const {i18n} = require('./next-i18next.config')
+
+const tsconfigPath = process.env.NEXTJS_TSCONFIG_PATH
+    ? process.env.NEXTJS_TSCONFIG_PATH
+    : './tsconfig.json'
```

Comment:
YAGNI, I'd rather keep PR small and introduce change incrementally as we need them